src/components/subcomponents/edit-status.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from "react";
+import { Grid } from "@material-ui/core";
+import Status from "./status";
+const statusLabel = [
+  "New Hack",
+  "Looking for members",
+  "In Progress",
+  "Team Closed",
+  "Canceled",
+  "Submitted"
+];
```

Comment:
you might want to define these as constants and use them consitantly across the files that care about status

```
export const Status = {
  NEW_HACK: "New Hack",
  // etc
}
```